ingestr/src/appsflyer/_init_.py

```diff
@@ -11,14 +13,38 @@
 def appsflyer_source(
     api_key: str, start_date: str, end_date: str
 ) -> Iterable[DltResource]:
+    start_date_obj = ensure_pendulum_datetime(start_date)
     client = AppsflyerClient(api_key)
 
-    @dlt.resource(write_disposition="merge", merge_key="install_time")
-    def campaigns() -> Iterable[TDataItem]:
-        yield from client.fetch_campaigns(start_date, end_date)
+    @dlt.resource(write_disposition="merge", merge_key="Install Time")
```

Comment:
the key is supposed to be as the fields in JSON, why did you go with `Install Time` instead of the previous version?

Reply:
Because "Install Time" is the JSON key not "install_time". My understanding was incorrect.